ddev/ddev
# add `DDEV_DATABASE_FAMILY` variable, fixes #4954

## The Issue

* #4954 

I want to generate database resource URLs for use in code and DDEV addons.

Currently, DDEV exposes `DDEV_DATABASE` which is  `app.Database.Type + ":" + app.Database.Version,`.

This is difficult to use because it must be parsed, and converted (MariaDB uses `mysql` protocol).

## How This PR Solves The Issue

This PR adds a new environmental variable: `DDEV_DATABASE_FAMILY`

- Defaults to `mysql`
- Set to `postgres` if database is `postgres`

## Manual Testing Instructions

1. Start a DDEV project
1. Type `ddev exec set | grep DDEV_DATABASE` to check currently set database variables

## Automated Testing Overview



## Related Issue Link(s)

Fixes #4954

## Release/Deployment Notes





<a href="https://gitpod.io/#https://github.com/ddev/ddev/pull/4991"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
